src/test/java/org/cryptomator/cryptofs/CryptoFileSystemProviderInMemoryIntegrationTest.java

```diff
@@ -58,6 +58,21 @@ public static void afterAll() throws IOException {
 		tmpFs.close();
 	}
 
+	@Test
+	@DisplayName("Replace an existing, shortened, empty directory")
+	public void testReplaceExistingShortenedDirEmpty() throws IOException {
+		try (var fs = setupCryptoFs(50, 100, false)) {
+			var dirName50Chars = "/target_89_123456789_123456789_123456789_123456789_"; //since filename encryption increases filename length, 50 cleartext chars are sufficient
+			var source = fs.getPath("/sourceDir");
+			var target = fs.getPath(dirName50Chars);
```

Comment:
setting up the fs and source/target can be done `@BeforeEach`, deduplicating some code shared between these two test cases

Reply:
It is intentional, that every test set up its own filesystems, not sharing a lot of coce. It would require an additional `@AfterEach` to close the filesystem and also I expect that this class will be populated with more, non-overlapping tests in #178.